cs/roadmap.html

```diff
@@ -145,6 +145,17 @@ <h3 class="title">General Algorithmic Improvements</h3>
 </div>
 
 
+<h3 class="title">General Improvements</h3>
```

Comment:
I would call these all "Software Development Improvements." The word "General" is too general.